modules/cms/classes/Theme.php

```diff
@@ -524,6 +524,17 @@ public function getCustomData()
         return ThemeData::forTheme($this);
     }
 
+    /**
+     * Remove data specific to this theme
+     * @return bool
+     */
+    public function removeCustomData()
+    {
+        if ($this->hasCustomData()) {
+            return $this->getCustomData()->delete();
+        }
+    }
```

Comment:
@DanHarrin please add a `return true` at the bottom of this method, as the docblock indicates a boolean return value.